xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -142,9 +142,9 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The read-only span to read.</param>
+        <summary>Reads an <see cref="T:System.Int32" /> out of a read-only span of bytes as little endian.</summary>
+        <returns>The little endian value.</returns>
         <remarks>To be added.</remarks>
```

Comment:
@ahsonkhan instead of adding a new remark to all this methods, would it be ok if I expand the summary to look like this?:

```xml
        <summary>Reads an <see cref="T:System.Int32" /> from the beginning of a read-only span of bytes as little endian.</summary>
```